text/src/autogluon/text/automm/configs/optimization/adamw.yaml

```diff
@@ -14,5 +14,4 @@ optimization:
   val_check_interval: 0.5
   top_k: 3
   top_k_average_method: 'greedy_soup'  # We support averaging method described in https://arxiv.org/pdf/2203.05482.pdf.
-                                       # Currently support "union_soup" and "greedy_soup".
-                                       # For other keys, the best model will be used.
+                                       # Currently support "union_soup", "greedy_soup", "max".
```

Comment:
Why called "best" in error but "max" in doc?

Reply:
It’s a typo.